routes/likeRoutes.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const app = express();
+const router  = express.Router();
+const {likeTweets , unLikeTweets} = require('../controllers/likeController')
+const { ensureAuthorizated } = require('../middeware/authorization')
+
+router.put('/likes/:id',ensureAuthorizated , likeTweets);
+
+router.put('/unlikes/:id',ensureAuthorizated , unLikeTweets);
```

Comment:
This should be a delete method.

And do you know from your app.js, this route would be `/likes/unlikes/:id/`?